tests/unit/ScreeningCard.spec.js

```diff
@@ -0,0 +1,58 @@
+import { mount } from "@vue/test-utils"
+import ScreeningCard from '@/components/screenings/ScreeningCard'
+
+import { getSeances } from "@/api/resources/SeancesRepository"
+
+const movie = {
+    id: 1,
+    title: "title",
+    genre: {
+        id: 1,
+        name: "genre"
+    },
+    poster_url: "poster.url",
+    length: 100,
+    release_date: "01.01.2001",
+    description: "description"
+}
+const seance = {
+    id: 1,
+    datetime: new Date,
+    movie: 1,
+    hall: 1
+}
+
+jest.mock('@/api/resources/SeancesRepository', () => {
```

Comment:
To be truly correct it would be nice to mock `@/api/FactoryRepository` because this is what you use inside the component, but it's definitely good enough as it is!